src/main/content/_assets/js/post.js

```diff
@@ -9,6 +9,11 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 
+const preTags = document.querySelectorAll('pre');
+preTags.forEach(preTag => {
```

Comment:
Since we currently use ES5 (a certain variety of JS), we cannot use the .forEach function.  Can you please modify this to be a standard for loop?